README.md

```diff
@@ -47,17 +47,17 @@ The log files can be found in `/var/log/rusk.log` and `/var/log/rusk-recovery.lo
 
 :information_source: To run the **latest release** of the Node Installer execute the following command:
 ```sh
-curl --proto '=https' --tlsv1.2 -sSfL https://github.com/dusk-network/node-installer/releases/download/v0.4.0/node-installer.sh | sudo sh
+curl --proto '=https' --tlsv1.2 -sSfL https://github.com/dusk-network/node-installer/releases/download/v0.5.0/node-installer.sh | sudo bash
 ```
```

Comment:
I think it would be nice if we could also have information & an example copy paste command in here for the different features which is now just the command for archive too.